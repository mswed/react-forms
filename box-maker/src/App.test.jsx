import { it, expect } from 'vitest';
import { render } from '@testing-library/react';
import App from './App';

it('Renders without crashing', async () => {
  render(<App />);
});

it('Matches snapshot', async () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
