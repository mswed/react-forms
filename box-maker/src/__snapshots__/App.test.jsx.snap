// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <form>
      <label
        for="width"
      >
        Width
      </label>
      <input
        id="width"
        name="width"
        placeholder="width"
        type="text"
        value=""
      />
      <label
        for="height"
      >
        Height
      </label>
      <input
        id="height"
        name="height"
        placeholder="height"
        type="text"
        value=""
      />
      <label
        for="color"
      >
        Color
      </label>
      <input
        id="color"
        name="color"
        placeholder="color"
        type="text"
        value=""
      />
      <button>
        Create Box
      </button>
    </form>
  </div>
</DocumentFragment>
`;
